projects/testing-library/tests/render.spec.ts

```diff
@@ -183,6 +185,73 @@ describe('componentOutputs', () => {
   });
 });
 
+describe('subscribeToOutputs', () => {
+  @Component({ template: ``, standalone: true })
+  class TestFixtureWithEventEmitterComponent {
+    @Output() readonly event = new EventEmitter<void>();
```

Comment:
It shouldn't matter (for now?), but could you also include a test using the new `output` here please?